apps/worker/src/app/shared/utils/cron-health.ts

```diff
@@ -1,12 +1,12 @@
 import { Logger } from '@nestjs/common';
 
-const url = 'https://uptime.betterstack.com/api/v1/heartbeat/';
+const url = process.env.CRON_CHECKIN_URL;
```

Comment:
i know it was my idea to extract it to the env variable, but I think that it will be nice to have a fallback to the origin URL as well just in case. Apologies for not raising it before i will push this small change if you don't mine so you won't have to. :)